FrontEnd/src/UseContext.jsx

```diff
@@ -0,0 +1,14 @@
+import { createContext, useContext, useState } from 'react';
+
+const ThemeContext = createContext();
+export const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState('light');
```

Comment:
Define an enum for theme values

Reply:
Could you please elaborate on what you mean here?

Reply:
I meant let's not hardcode strings like `light`. Define all supported theme values as an enum or object and import from there.